backend/src/app.ts

```diff
@@ -0,0 +1,34 @@
+import express, { json } from "express";
+import config from "config";
+import sequelize from "./db/sequelize";
+import errorLogger from "./middlewares/error/error-logger";
+import errorResponder from "./middlewares/error/error-responder";
+import notFound from "./middlewares/not-found";
+import cors from "cors";
+import serverRouter from "./routers/server";
+import serverCompanyRouter from "./routers/serverCompany";
+
+const port = config.get<string>("app.port");
+const name = config.get<string>("app.name");
+const force = config.get<boolean>("sequelize.sync.force");
+
+const app = express();
+
+(async () => {
+  await sequelize.sync({ force });
+
+  // middlewares
+  app.use(cors()); // allow any client to use this server
+
+  app.use(json()); // a middleware to extract the post/put/patch data and save it to the request object in case the content type of the request is application/json
+
+  app.use("/servers", serverRouter);
+  app.use("/serverCompanys", serverCompanyRouter);
+  app.use(notFound);
```

Comment:
all in all the backend works, however the update doesn't, and, this server isnt actually doing what it is intended to do according to the spec